### ISSUE TITLE
In Sublime Text v 3.0 build 3143 switching colours via menu lead to error

### ISSUE THREAD

**Issue by wiiiimm** [chmln/sublime-text-theme-switcher-menu#11]
Once this plugin changes the Preferences.sublime-settings, 
color scheme will go crazy and 'Error loading colour scheme' will keep popping up.
the color scheme need to be resetted by deleting "color_scheme" entry or by reselecting another theme via the command + p + shift command.

**Comment by chmln** [comment]
Hello @williamli 

I can't reproduce this in the latest build.
Perhaps other plugins could be interfering?

**Comment by monfresh** [comment]
I can confirm this is an issue with the latest version of Sublime Text, 3.2.2 build 3211 on macOS Catalina 10.15.7. 

I started with a brand new installation of Sublime Text, where `Preferences->Color Scheme` does not have a submenu. I then installed the Base16 Color Schemes package, and I was able to change themes by selecting `Preferences->Color Scheme` from the menu bar, which then displays a dropdown within Sublime Text, where I can see a live preview of each theme as I use the arrow keys.

After installing Themes Menu Switcher, it added a submenu inside `Preferences->Color Scheme` which shows all the currently installed color schemes. If I try to select any theme inside `Preferences->Color Scheme->Base16 Color Schemes`, then I get an error that it could not load it, such as: `Error loading colour scheme base16-atelier-plateau.tmTheme: Unable to open base16-atelier-plateau.tmTheme`.

<img width="532" alt="Screen Shot 2020-11-15 at 4 09 52 PM" src="https://user-images.githubusercontent.com/811150/99196899-1e604700-275d-11eb-8c16-9ad4d2eeb8ec.png">

If instead, I use `shift-command-p` and then `UI: Select Color Scheme`, then it works fine.
